dev/bots/check_code_samples.dart

```diff
@@ -416,6 +416,7 @@ final Set<String> _knownMissingTests = <String>{
   'examples/api/test/widgets/transitions/animated_widget.0_test.dart',
   'examples/api/test/widgets/transitions/slide_transition.0_test.dart',
   'examples/api/test/widgets/transitions/listenable_builder.2_test.dart',
+  'examples/api/test/widgets/transitions/matrix_transition.0_test.dart',
```

Comment:
This known missing test list is in the `check_code_sample.dart` file is for existing examples and its goal is to add missing tests for the existing examples.

Could you please add a test for your example?  Since your example is new.

Reply:
Thanks for pointing this out. Since the other animation transitions were missing tests, I assumed it wasn't practical for this one, but I saw some other tests I could get inspiration from.

The test I added is a bit small however. I wanted to add some checks for the actual transform, but I ran into trouble because the example app as its own AnimationController, and the test needs to provide its own to properly test animations.

I could add an AnimationController to the example app constructor, and make the test more thorough this way, but at the expense of slightly increased complexity in the ExampeApp. Please tell me if the way I did it now looks good to you, you whether you'd rather go the more thorough route.